WordPress/src/main/java/org/wordpress/android/ui/themes/ThemeBrowserActivity.java

```diff
@@ -328,6 +328,10 @@ private void fetchInstalledThemesIfJetpackSite() {
     }
 
     private void activateTheme(String themeId) {
+        ThemeActivationBottomSheetFragment bottomSheet = new ThemeActivationBottomSheetFragment();
+        bottomSheet.show(getSupportFragmentManager(), ThemeActivationBottomSheetFragment.TAG);
```

Comment:
Thank you for adding this 🙇 
This will help with testing this PR individually. Please also add test steps in the PR description if possible.
Though this seems trivial at this point the PR description will provide a good source of documentation if we ever get back to this code in the future. You can check previous PRs in the repository to get some examples of PR descriptions.

Reply:
Done